README.md

```diff
@@ -0,0 +1,29 @@
+# fobnail
+
+## Usage
+
+TBD
+
+## Development
+
+### pre-commit hooks
+
+* Install [rustfmt](https://github.com/rust-lang/rustfmt) and
+  [rust-clippy](https://github.com/rust-lang/rust-clippy)
+
+* [Install pre-commit](https://pre-commit.com/index.html#install)
+
+* Install hooks into repo:
+
+```
+pre-commit install
+```
```

Comment:
This puts significant software requirements on users (Python and Rust). Since we're using docker anyway for building, would it be possible to put those inside `fobnail-sdk` and start a container from `pre-commit` hook?

Reply:
@krystian-hebel Got working pre-commit. Requires latest SDK https://github.com/fobnail/fobnail-sdk/pull/9
See README.md for instructions.